### PR TITLE
analytics data model per db for better querying

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: src/config/ci.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { ConfigInterface, } from './config';
+
+const config: ConfigInterface = {
+  http: {
+    port: 8088,
+  },
+  modules: {
+    latestVersion: '0.0.7',
```
Comment (aguillenv) [inline src/config/ci.ts:8]: Hmm probably this should be already 0.0.8?

File: package.json [hunk 2]
```diff
@@ -16,9 +16,7 @@
     "start:dev:watch": "yarn build && yarn serve",
     "gen-sql": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:generate -n",
     "migrate": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:run",
-    "gen-module": "IASQL_ENV=local ./.module_migration.sh",
-    "docker-compose:dev": "IASQL_ENV=local docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build",
-    "docker-compose": "IASQL_ENV=local docker-compose up --build"
```
Comment (aguillenv) [inline package.json:21]: Not sure about removing this one. Are we sure this is not the one in use in the `CONTRIBUTING.md`?

File: src/config/index.ts [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import { ConfigInterface, throwError, } from './config';
 
+export const ENV = process.env.IASQL_ENV;
```
Comment (aguillenv) [inline src/config/index.ts:3]: Maybe `IASQL_ENV`?

File: src/config/production.ts [hunk 4]
```diff
@@ -34,7 +34,7 @@ const config: ConfigInterface = {
     release: process.env.SENTRY_RELEASE ?? throwError('No Sentry Release defined'),
   },
   telemetry: {
-    amplitudeKey: '968524573d0f8bd2e84460099bca9353'
+    amplitudeKey: '8fd6aa1d61c115e59e35b3adfd5dd41a'
```
Comment (aguillenv) [inline src/config/production.ts:37]: Is this the same as local? How did we said we could differentiate them if they are in the same dashboard? Or we do mot need to?

File: src/services/telemetry.ts [hunk 5]
```diff
@@ -11,33 +11,33 @@ type IasqlOperationType = typeof IasqlOperationType[keyof typeof IasqlOperationT
 
 const singleton = config.telemetry ? Amplitude.init(config.telemetry.amplitudeKey) : undefined;
 
-export async function logDbConnect(dbId: string, dbAlias: string, uid: string, email: string) {
+export async function logDbConnect(dbId: string, userProps: any) {
```
Comment (aguillenv) [inline src/services/telemetry.ts:14]: nit: maybe add `todo` to type this

**Review by aguillenv — APPROVED** [review]

File: src/config/production.ts (hunk 4 above)
Comment (depombo) [reply src/config/production.ts:37]: yes it is the same as local. we can differentiate by environment

File: src/services/telemetry.ts (hunk 5 above)
Comment (depombo) [reply src/services/telemetry.ts:14]: hmm it varies per event, but maybe I can type it now

File: src/config/ci.ts (hunk 1 above)
Comment (depombo) [reply src/config/ci.ts:8]: ah yes

File: package.json (hunk 2 above)
Comment (depombo) [reply package.json:21]: This is what we use in contributing. Removing this mostly for us, but will add back

**Pull request merged**